### DISCUSSION TITLE
WiGui detected as malware by Bitdefender

### DISCUSSION THREAD

**Discussion by lukas-0506** [Romanitho/Winget-Install-GUI#71] [General]
We're looking for an alternative to the Software Center delivered with SCCM and I found WiGui. I tried it on my private PC and got this message.

![image](https://user-images.githubusercontent.com/61327758/203394276-e42bd789-4e3d-46a9-9bc6-4040bd2d496c.png)

So for now, we'll stay away from his program.

**Comment by KnifMelti** [comment]
Thats because the .ps1 is compiled with https://github.com/MScholtes/PS2EXE - read on his repo why it's reported as false positive by some AV.
You could always run the Winget-Install-GUI.ps1 directly or run it without download:
https://github.com/Romanitho/Winget-Install-GUI#run-without-download

**Reply by Romanitho** [reply]
Yes, feel free to use native powershell script (as describe at the end or README.md). Read it to be sure we haven't put bad things in it 😇 and execute without EXE, as some AV can detect that as bad thing.

**Comment by KnifMelti** [comment]
Old now, but still relevant; in a commit from Sep 19, 2020 [README.md](https://github.com/MScholtes/PS2EXE/commit/1b4f17236f36c80a2e797b5b48ac64ea2ae8f3da) we were told (nowadays not included):

> Attention: Incorrect virus detection of PS2EXE
> 
> Some stupid idiot seems to have abused PS2EXE to compile his computer virus script. As a result, a rapidly growing number of virus scanners recognize programs created with PS2EXE as malicious programs and delete them.
> 
> There is only one hope to save the PS2EXE project: Please send your (harmless) programs created with PS2EXE via the web forms from the virus scanners' vendors for reporting false positives (I've already done it with some of them, please use only the false positive page)!
> 
> If this is not successful, then I will have to quit PS2EXE as nobody can use it anymore.

**Reply by Romanitho** [reply]
I agree
